### PR TITLE
Fix: Social Icons in Connection Sharing

### PR DESCRIPTION
**Pull request by enejb**
This Pr Adds nextdoor icon. 
+ it uses the social icons from the dashboard in this list. 

Before:

After:

To test:

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/Classes/ViewRelated/Blog/SharingViewController.m [hunk 1]
```diff
@@ -295,9 +295,8 @@ - (void)configurePublicizeCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath
     }
 
     // Configure the image
-    UIImage *image = [WPStyleGuide iconForService: publicizer.serviceID];
+    UIImage *image = [WPStyleGuide socialIconFor:publicizer.serviceID];
     [cell.imageView setImage:image];
-    cell.imageView.tintColor = ([connections count] > 0) ? [WPStyleGuide tintColorForConnectedService: publicizer.serviceID] : [UIColor murielListIcon];
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Blog/SharingViewController.m:300]: Please remove unused tintColorForConnectedService.

Comment (kean) [reply WordPress/Classes/ViewRelated/Blog/SharingViewController.m:300]: It's still used in `PostSettingsViewController`. Let's make sure this change doesn't affect this screen and remove it in this screen to if possible. If not, it's good to merge 👍

**Review by kean — APPROVED** [review]

File: WordPress/Classes/ViewRelated/Blog/SharingViewController.m (hunk 1 above)
Comment (enejb) [reply WordPress/Classes/ViewRelated/Blog/SharingViewController.m:300]: Will do this in a different PR when we remove the remote feature flag.

**Pull request merged**